### DISCUSSION TITLE
LabWC distro?

### DISCUSSION THREAD

**Discussion by glow12121** [labwc/labwc#2312] [Q&A]
Is there a ready-made distro solution featuring LabWC? Something with a clear simple interface?

I need to install lightweight OS on a laptop that belongs to a lady in her 60s. For the OS to be future-proof, I want to go for Wayland.

I've stumbled upon the news about Raspberry Pi OS adopting LabWC - that's excellent, but alas this OS is not available for 32 or 64-bit PC architecture.

**Comment by Narrat** [comment]
As far as I know there is no such distro available for x86 systems as of now.
But it should be relatively easy for you to adopt the work of RPI OS onto the distribution which you're most comfortable with.

**Reply by glow12121** [reply]
Her system is actually 64 bit. I ended up installing 64-bit Windows 7.

**Comment by fungilife** [comment]
https://sourceforge.net/projects/joborun/files/images/ 

This is a screenshot from joborun-linux labwc setup

https://a.fsdn.com/con/app/proj/joborun/screenshots/labwc-0.8.0-rc29sm-ebb777c0.png

**Reply by glow12121** [reply]
Thank you. It looks like a hacker's desktop though while I need something user-friendly.

**Comment by stefonarch** [comment]
Most probably debian testing will include LXQt 2.1 which can be easily combined with labwc, just setting labwc as compositor in "Session Settings".
I wouldn't go for any other distro in that case.

**Reply by glow12121** [reply]
Thank you!

**Comment by sfs-pra** [comment]
[Lab Remix (Ubuntu 24.04 wayland: labwc+sfwbar )](https://forum.runtu.org/index.php/topic,9005.msg60155.html#msg60155)

**Comment by probonopd** [comment]
Wonder what happened to the Raspberry Pi Desktop for PC and Mac. 

Hopefully they will update it.

https://www.raspberrypi.com/software/raspberry-pi-desktop/

**Reply by zenfas** [reply]
Raspberry Pi Desktop for PC no update from 2022, if they release new one, I will try it

**Comment by sfs-pra** [comment]
[TrixiePup64 Wayland 11.0.0 Beta-2](https://forum.puppylinux.com/viewtopic.php?t=14554)
